app/src/androidTest/java/org/projectbuendia/client/ui/FunctionalTestCase.java

```diff
@@ -437,4 +441,37 @@ protected void inPatientListClickPatientWithId(String id) {
             .inAdapterView(hasId(R.id.fragment_patient_list))
             .atPosition(0));
     }
+
+    /** Generates IDs to identify the newly created patient. */
+    protected String generateId() {
+        String id = Long.toString(new Date().getTime()%100000);
+        return id;
```

Comment:
id is only used once; simplify to just `return <expr>`

`"" + x` is more concise than `Long.toString(x)`